src/operations/client_bulk_write/command_builder.ts

```diff
@@ -398,6 +400,9 @@ function createUpdateOperation(
   if (model.collation) {
     document.collation = model.collation;
   }
+  if (!multi && 'sort' in model && model.sort != null) {
```

Comment:
I don't see any tests that demonstrate that `sort` isn't supported for updateMany, either through the client-level API or collection-level API.  Can we add these tests?

Reply:
done